### ISSUE TITLE
Now Hiring: OpenRefine Developer & Contributor Engagement

### ISSUE THREAD

**Issue by magdmartin** [OpenRefine/OpenRefine#7037]
I am cross-posting our job listing for the OpenRefine Developer & Contributor Engagement role here. For all the details, including the application process, please visit [our blog.](https://openrefine.org/blog/2024/11/25/openrefine-developer-role)

**Comment by devatbosch** [comment]
Hi @magdmartin ,

Hope you are doing good.

Wanted to know is the indicated role still open? If so can I be given a chance to apply for the process...

Thank you!

**Comment by magdmartin** [comment]
@devatbosch Yes, applications are still open. I received your application and we will review it. 
thank you
